pkg/common/helpers_test.go

```diff
@@ -16,14 +16,71 @@ package common
 import (
 	"reflect"
 	"testing"
+	"time"
 
 	wfv1 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
-	"github.com/onsi/gomega"
-
+	wfv1versioned "github.com/argoproj/argo-workflows/v3/pkg/client/clientset/versioned"
 	addonv1 "github.com/keikoproj/addon-manager/api/addon/v1alpha1"
+	addonv1versioned "github.com/keikoproj/addon-manager/pkg/client/clientset/versioned"
+	"github.com/onsi/gomega"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/rest"
 )
 
+func TestGetCurretTimestamp(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+	timestamp := GetCurretTimestamp()
+	expectedTime := time.Unix(0, timestamp*int64(time.Millisecond))
```

Comment:
Let's use the raw `timestamp` value and make expectedTime something like this?
```suggestion
	expectedTime := time.Now().UnixMilli()
```

Reply:
if I make that change, expectedTime will be int64 type. ```func gomega.BeTemporally(comparator string, compareTo time.Time, threshold ...time.Duration) types.GomegaMatcher``` takes in time.Time type for the compareTo paramter. So, I made expectedTime as the same type time.Time